### ISSUE TITLE
axios fails on IE8

### ISSUE THREAD

**Issue by dmatteo** [mzabriskie/axios#152]
The error is the usual IE8 generic `Object doesn't support this property or method`, which points to this line:

```js
 return _axios2['default'].get(url).then(function (response) {
```

Which maps to this line in my source code
```js
return axios.get(url).then((response) => {
```

I have all the polyfills in place, but still axios won't work.

Any idea before I fallback to XHR?

**Comment by mzabriskie** [comment]
Can you try debugging a bit to see where it is failing?

```js
// Prefer console.log, but you're using IE8 :-P
alert(typeof axios.get);
alert(typeof axios.get(url).then);
alert(typeof Promise);
```

**Comment by ctimmerm** [comment]
This should be fixed by https://github.com/mzabriskie/axios/commit/1e2cb9bdca0e19298aa7c242a7db4b45427967ea

**Comment by dmatteo** [comment]
Sorry I haven't had the chance to try and debug this, but if you think this may work I'm happy to try it once released

**Comment by mzabriskie** [comment]
Just released [0.8.0](https://github.com/mzabriskie/axios/blob/master/CHANGELOG.md#080-dec-11-2015). Closing for now. Please re-open if you still see this problem.
